modules/call-dispatch/src/lib.rs

```diff
@@ -194,11 +202,11 @@ impl<T: Trait<I>, I: Instance> MessageDispatch<T::MessageId> for Module<T, I> {
 
 		// prepare dispatch origin
 		let origin_account = match message.origin {
-			CallOrigin::BridgeAccount => bridge_account_id(bridge, CALL_DISPATCH_MODULE_PREFIX),
-			CallOrigin::RealAccount(source_public, target_public, target_signature) => {
+			CallOrigin::SourceRoot => derive_target_account_id(bridge, T::SourceChainAccountId::default()),
```

Comment:
Can we prefix this somehow, so there is no collision with `CallOrigin::SourceAccount`?